### ISSUE TITLE
seach player error

### ISSUE THREAD

**Issue by SergitoKLK** [ESX-Org/esx_policejob#456]
Hello people I have a problem with esx_police

My problem is that it does not allow to search for players or see their identity but it does not report an error, here is the video

Sorry for my English

Identidad = Identity
Buscar = Seach

Video:
https://drive.google.com/file/d/1tG51te8pUIGExJAQYhowAjKVNm6caih9/view?usp=sharing

Thanks for reading

**Comment by SergitoKLK** [comment]
I have found the error, is this
SCRIPT ERROR @esx_policejob/client/main.lua:1493 attemp to index a nil value (global 'esx')

**Comment by FBFezz** [comment]
Find a fix for this yet?

**Comment by SergitoKLK** [comment]
I still haven't found it, still don't let me confiscate objects

**Comment by Parity-lab** [comment]
same problem here

**Comment by bosszalapee** [comment]
same problem

**Comment by SergitoKLK** [comment]
do you already have the solution?

**Comment by FBFezz** [comment]
Yeah i updated to latest version of esx (I'm using extendedmode but its the same as es_extended)
